platform/containers/axclustermanager/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM %%ARGO_BASE_REGISTRY%%/argobase/kubernetes-distro:ax-v1.6.7-6341-095136c-050831f
+FROM %%ARGO_BASE_REGISTRY%%/argobase/kubernetes-distro:v1.6.7
 
 # Remove built-in kubeup code and copy AX version to container.
 RUN rm -rf /kubernetes/cluster
 ADD docker_build/cluster /kubernetes/cluster
 
-ENV AX_KUBE_VERSION ax-v1.6.7-6341-095136c-050831f
+ENV AX_KUBE_VERSION ${KUBERNETES_VERSION}
```

Comment:
${KUBERNETES_VERSION} is an env variable. Will this get substituted when we run docker build?

Reply:
@shrinandj it's set in base image

Reply:
@shrinandj good question. I had doubt and tested it. It worked.